src/lib/y2partitioner/sequences/add_partition.rb

```diff
@@ -8,15 +8,29 @@
 
 module Y2Partitioner
   module Sequences
+    # Collecting params of partition to be created
+    # and remembering them across dialogs
+    class PartitionTemplate
```

Comment:
JFYI actually it is not business of ruby documentation to explain it. I found https://github.com/lsegal/yard/issues/3, but it is only auto-generation. Which can be customized only with plugin - https://github.com/michaeledgar/yard-struct